dist/21P-530-schema.json

```diff
@@ -4,6 +4,42 @@
   "type": "object",
   "additionalProperties": false,
   "definitions": {
+    "address": {
+      "type": "object",
+      "properties": {
+        "street": {
+          "type": "string"
+        },
+        "aptNum": {
+          "type": "string",
+          "minLength": 1,
+          "maxLength": 5
+        },
+        "city": {
+          "type": "string"
```

Comment:
for the longer fields i have overflow code that will put "see additional information page" in the field and add an extra page with the answer if it goes over the limit, technically we could use this for all the fields but there's no space to put the "see last page" text for fields that have <5 characters available so i would have to make a custom solution for this section where all the address fields are moved to the additional info page and it would be a lot more complicated for me, we could do it if it was really important to  support addresses that don't fit the form though